fast64_internal/oot/oot_level_writer.py

```diff
@@ -258,7 +258,8 @@ def readSceneData(scene, scene_properties, sceneHeader, alternateSceneHeaders):
     scene.naviCup = getCustomProperty(sceneHeader, "naviCup")
     scene.skyboxID = getCustomProperty(sceneHeader, "skyboxID")
     scene.skyboxCloudiness = getCustomProperty(sceneHeader, "skyboxCloudiness")
-    scene.skyboxLighting = getCustomProperty(sceneHeader, "skyboxLighting")
+    scene.skyboxLighting = sceneHeader.skyboxLighting
+    scene.skyboxLightingCustom = sceneHeader.skyboxLightingCustom
```

Comment:
Sorry, I'm not a fan of this - the `OOTScene` is supposed to be a representation of the scene in C, so `skyboxLighting` shouldn't include details of the property implementation in Blender. Since this is only for comments I would prefer just simplifying the comments to not rely on this, but if you want to keep it an alternative would be to add something like an `isSkyboxLightingCustom` boolean to handle the comments, while retaining `getCustomProperty` for `skyboxLighting.`

Reply:
chose the bool method as it was easier to do